docs/source/rabbitmq-tutorial/examples/2-work-queues/new_task_initial.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+import sys
+
+from aio_pika import DeliveryMode, Message, connect
+
+
+async def main() -> None:
+    # Perform connection
+    connection = await connect("amqp://guest:guest@localhost/")
+
+    async with connection:
+        # Creating a channel
+        channel = await connection.channel()
+
+        message_body = b" ".join(
+            arg.encode() for arg in sys.argv[1:]
+        ) or b"Hello World!"
+
+        # Sending the message
+        await channel.default_exchange.publish(
```

Comment:
```python
confirmation = await channel.default_exchange.publish(
```

Reply:
Then `pylama` will scream because of the unused variable. What shall I do with the `confirmation`?
I tried to do something useful with it like reading the `delivery_tag`, but this is how i encountered the unexpected `DeliveredMessage` by accident. (mosquito/aiormq#183)